atlassian/confluence.py

```diff
@@ -1952,6 +1952,27 @@ def add_user_to_group(self, username, group_name):
         data = {'name': username}
         return self.post(url, params=params, data=data)
 
+    def add_space_permissions(self, space_key, subject_type, subject_id, operation_key, operation_target):
+        """
+        Add permissions to a space
+
+        :param space_key: str
+        :param subject_type: str
+        :param subject_id: str
+        :param operation_key: str
+        :param operation_target: str
+        :return: Current permissions of space
+        """
+        url = 'rest/api/space/{}/permission'.format(space_key)
+        data = {
+            'id': 2154,
```

Comment:
Hi!
looks like id is hardcoded :)

Reply:
Yes, at one point when debugging (I think the problem was I was unnecessarily wrapping the data in extra layer of JSON), I put the id from the API example in.

I just tested with it removed, and it worked, so let me fix that.